client/src/Error.js

```diff
@@ -0,0 +1,26 @@
+
+//Add route for error page...
+// <Route component = {error}>
+// import Logo from '../logo.png';
+import React from "react";
+import {NavLink} from "react-router-dom";
+import Logo from './design/error-screen/sreens/error-sreen-desktop.png'
```

Comment:
To add an Image, create a directory called `assets`, in the `src`, add the image there and reference it from there.